includes/wikia/models/DesignSystemFooterModel.class.php

```diff
@@ -0,0 +1,429 @@
+<?php
+
+class DesignSystemFooterModel extends WikiaModel {
```

Comment:
In all our RFCs we use words "Global Footer". Do you think it makes sense to keep naming consistent here?